lib/block-ref-grammar.ym

```diff
@@ -24,6 +24,7 @@
 
 %code requires {
 
+#pragma GCC diagnostic ignored "-Wswitch-default"
```

Comment:
For now `--enable-extra-warnings` is turned off for clang, but I would like to avoid diversion between clang and gcc.
As far as I found out clang support gcc pragmas, but can you verify it too please?
Here is the documentation I found about clang and pragmas
https://clang.llvm.org/docs/UsersManual.html#controlling-diagnostics-via-command-line-flags

Reply:
Yes @gaborznagy. Clang supports gcc pragmas.

Thus, we should add one more line as ```#pragma clang diagnostic ignored "-Wswitch-default"```.

May I do the this change and commit ?

Reply:
As clang support gcc pragmas we should not add an extra line with a clang pragma.
The second option you suggested should be used only if you would like to ignore warnings for specific lines instead of the whole file.

So what I suggest is to have gcc pragmas only.

Reply:
Ok. I got the point.